src/components/RadioInputBox/OptionInputs.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from "react";
+import { useFormikContext, Field } from "formik";
+import classnames from "classnames";
+
+import DescriptionSummary from "./DescriptionSummary";
+
+type Options = OptionDetails[];
+
+type OptionDetails = {
+  name: string;
+  description: string;
+  type: string;
+  required: boolean;
+};
+
+type OptionValue = {
+  [key: string]: string;
+};
+
+type Props = {
+  name: string;
+  options: Options;
+  onValuesChange: (name: string, values: OptionValue) => void;
```

Comment:
It could but that type is defined in the panel and this is a generic component so I didn't want them to be dependent. We also don't have many cases where shared types like this are required so I figured a duplicate of type would be ok in this situation.